luajit/luajit.json

```diff
@@ -10,14 +10,14 @@
     ],
     "sources": [
         {
-            "type": "archive",
-            "url": "https://luajit.org/download/LuaJIT-2.1.0-beta3.tar.gz",
-            "sha256": "1ad2e34b111c802f9d0cdf019e986909123237a28c746b21295b63c9e785d9c3",
+            "type": "git",
+            "url": "https://github.com/openresty/luajit2.git",
+            "tag": "v2.1-20231117",
+            "commit": "4182d6bf37e9f8d1cb5d6e83b1db66de84b95101",
             "x-checker-data": {
-                "type": "anitya",
-                "project-id": 6444,
-                "stable-only": false,
-                "url-template": "https://luajit.org/download/LuaJIT-$version.tar.gz"
+                "type": "git",
+                "tag-pattern": "^v([\d.-]+)$",
```

Comment:
Did you test this with the data checker? I assume you need to escape `\\`

Reply:
Whoops! Sorry! 

Good catch, I had eyeballed the regex but I guess my regex-fu is not like it used to be..

Fixed and tested with flatpak-external-data-checker to be sure it is working.

Reply:
Just double escaping things because of JSON, thanks for testing.